packages/input-output-logger/index.js

```diff
@@ -9,7 +10,8 @@ module.exports = (opts) => {
   const { logger, omitPaths } = Object.assign({}, defaults, opts)
 
   const omitAndLog = message => {
-    const redactedMessage = omit(message, omitPaths)
+    const messageClone = cloneDeep(message)
```

Comment:
Can you test using `JSON.parse(JSON.stringify(o))` instead. `_.cloneDeep` is notoriously slow in comparison, unless you're using the additional edge cases.

https://github.com/lodash/lodash/issues/1984

Reply:
Updated and tested, thanks for your comment!